### PR TITLE
Add info about all-fqans option to LCMAPS VOMS docs

### PR DESCRIPTION
**Pull request by matyasselmeci**
SOFTWARE-2932

### CODE REVIEW

File: docs/security/lcmaps-voms-authentication.md [hunk 1]
```diff
@@ -208,6 +209,21 @@ Each non-commented line is a shell-style pattern which is compared against a use
...
+### Mapping using all FQANs
+
+By default, the LCMAPS VOMS plugin only considers for mapping the first FQAN of a VOMS proxy. This behavior matches what GUMS does. If you want to consider all FQANs, you must set the appropriate option.
+
+-   If you are using osg-configure, set `all_fqans = True` in `10-misc.ini`, then run `osg-configure -c`
+
+-   If you are configuring `lcmaps.db` manually (see [manual configuration](#manual-configuration) below), add `"-all-fqans"` to the module definitions for `vomsmapfile` and `defaultmapfile`
+
+!!! note
+    If you are using OSG 3.3, osg-configure should be at least version 1.10.2.  If you are using OSG 3.4, osg-configure should be at least version 2.2.2.
```
Comment (brianhlin) [inline docs/security/lcmaps-voms-authentication.md]: This note should go at the top of the section

File: docs/security/lcmaps-voms-authentication.md [hunk 2]
```diff
@@ -208,6 +209,21 @@ Each non-commented line is a shell-style pattern which is compared against a use
 !!!warning
     `/etc/grid-security/ban-mapfile` *must* exist, even if you are not banning any users. In that case, the file should be blank. If the file does not exist, LCMAPS will ban every user.
 
+
+
+### Mapping using all FQANs
+
+By default, the LCMAPS VOMS plugin only considers for mapping the first FQAN of a VOMS proxy. This behavior matches what GUMS does. If you want to consider all FQANs, you must set the appropriate option.
```
Comment (brianhlin) [inline docs/security/lcmaps-voms-authentication.md]: > ... only considers the first FQAN of a VOMS proxy for mapping

Comment (brianhlin) [reply docs/security/lcmaps-voms-authentication.md]: > ... matches the behavior of GUMS

File: docs/security/lcmaps-voms-authentication.md (hunk 1 above)
Comment (matyasselmeci) [reply docs/security/lcmaps-voms-authentication.md]: Seems like a non sequitur if I put it there. What if I put it right after the bullet about osg-configure?

Comment (brianhlin) [reply docs/security/lcmaps-voms-authentication.md]: Works for me

**Pull request merged**